src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1364,7 +1364,10 @@ private function addHttpClientSection(ArrayNodeDefinition $rootNode)
                                     ->info('A comma separated list of hosts that do not require a proxy to be reached.')
                                 ->end()
                                 ->floatNode('timeout')
-                                    ->info('Defaults to "default_socket_timeout" ini parameter.')
+                                    ->info('The inactivity timeout. Defaults to "default_socket_timeout" ini parameter.')
```

Comment:
Added this info that I think is useful.

Reply:
added to  #32928, please revert from here